scripts/data/common/killed-in-gaza/reconcile_lists.ts

```diff
@@ -106,6 +108,7 @@ const flipDateParts = (dob: string) => {
   return `${year}-${month}-${date}`;
 };
 
+// TODO: how is this Ref Date determined
```

Comment:
this was the date of the initial list release i believe, so we kept it b/c it was representative of the bulk of the list at the time

probably OK to keep for now as it's close to the "midpoint" of the timeframe so far...